ui/js/components/auth.js

```diff
@@ -32,7 +32,7 @@ treeherder.component("login", {
            ng-click="$ctrl.login()">Login/Register</a>
         <span ng-if="!$ctrl.userCanLogin"
               class="midgray"
-              title="thServiceDomain does not match host domain">Login not available</span>
+              title="thServiceDomain does not match host domain or already logging in">Login not available</span>
```

Comment:
While logging on on this, it looks kind of weird to have it say "Login not available" during login.  Would you make a second ``ng-if`` span here that does an if on ``$ctrl.userLogginIn`` or something like that and have it say "Logging in..."?